components/slider/slider.ts

```diff
@@ -66,6 +70,15 @@ export class Slider implements AfterViewInit,OnDestroy,OnChanges,ControlValueAcc
                     this.onChange.emit({originalEvent: event, values: ui.values});
                 }
                 else {
```

Comment:
If value of slide exceeds limitedMin or limitedMax, set ui value and slider handle value to limitedMin/Max and prevent slide from occurring past these points